```
pandas/core/arrays/string_arrow.py:113: error: Return type "Type[ArrowStringArray]" of "construct_array_type" incompatible with return type "Type[StringArray]" in supertype "StringDtype"  [override]
pandas/core/arrays/string_arrow.py:129: error: Return type "ArrowStringArray" of "__from_arrow__" incompatible with return type "StringArray" in supertype "StringDtype"  [override]
```

--- a/pandas/core/arrays/string_arrow.py
+++ b/pandas/core/arrays/string_arrow.py
@@ -126,9 +126,9 @@ class ArrowStringDtype(StringDtype):
     def __repr__(self) -> str:
         return "ArrowStringDtype"
 
-    def __from_arrow__(
+    def __from_arrow__(  # type: ignore[override]
         self, array: pa.Array | pa.ChunkedArray
-    ) -> ArrowStringArray:  # type: ignore[override]
+    ) -> ArrowStringArray:
         """
         Construct StringArray from pyarrow Array/ChunkedArray.
         """

--- a/pandas/core/arrays/string_arrow.py
+++ b/pandas/core/arrays/string_arrow.py
@@ -110,7 +110,7 @@ class ArrowStringDtype(StringDtype):
         return str
 
     @classmethod
-    def construct_array_type(cls) -> type_t[ArrowStringArray]:
+    def construct_array_type(cls) -> type_t[ArrowStringArray]:  # type: ignore[override]
         """
         Return the array type associated with this dtype.
 
@@ -126,7 +126,9 @@ class ArrowStringDtype(StringDtype):
     def __repr__(self) -> str:
         return "ArrowStringDtype"
 
-    def __from_arrow__(self, array: pa.Array | pa.ChunkedArray) -> ArrowStringArray:
+    def __from_arrow__(
+        self, array: pa.Array | pa.ChunkedArray
+    ) -> ArrowStringArray:  # type: ignore[override]
         """
         Construct StringArray from pyarrow Array/ChunkedArray.
         """

